### ISSUE TITLE
10周年なのでそろそろリニューアルしたい

### ISSUE THREAD

**Issue by jacoyutorius** [hamamatsu-rb/hamamatsu-rb.github.com#18]
## 現行機能の整理

- イベント情報
  - いる
- 過去のイベント, 浜松Ruby会議01を開催しました
  - いる？
- メーリングリスト
  - いらない
- Twitter
  - いる 
- Facebook
  - いる
- WIki へのリンク
  - いる
- メーリングリストへのリンク
  - いる？

## Slack へのinvite

メーリングリストの代わりにSlack へのinviteを行うようにしたい。
（JAWS-UGではGoogle Spread SheetのFormに入力することでSlackから自動的にinviteメールが飛ぶようになっている）

## 機能要件

Githubにpushしたら自動的に公開されるようにしたい。

- 現行の GIthubPages 使う
- 今っぽくJAM Stackにする
  - Netrify とか AWS Amplifyとか？
  - git push => サーバーレスでhtmlビルド => Github Pages / ホスティングサーバーへデプロイ って感じ？

**Comment by Langur** [comment]
- 過去のイベントは浜松IT系合同〇〇を追加しても良いかも。あとはOSCに出展しましたとか。
- メーリングリストは表示しないならリンクも不要な気がします。
